.travis.yml

```diff
@@ -8,13 +8,18 @@ before_install:
 - git remote set-branches origin master && git fetch
 - git checkout master
 - git checkout -
+
+before_script:
+- npm run nsp-check # -- --warn-only
```

Comment:
Ah, actually I left that in.
The way nsp+CI works is, once a new vulnerability is found it may fail your CI, and your Travis will remain broken until a newer version of that vulnerable dependency is released. This could take hours/days/weeks. So we'd have two obvious solutions:

- Uncomment the `-- --warn-only` bit, so nsp still runs, but won't exit with an exit code !== 0.
- Add the vulnerable module exception to the .nsprc file so we essentially ignore the warning.

I'm happy to remove the comment above, but just wanted to explain what the comment was for.